hathor/transaction/resources/block_at_height.py

```diff
@@ -75,8 +75,29 @@ def render_GET(self, request: 'Request') -> bytes:
             })
 
         block = self.manager.tx_storage.get_transaction(block_hash)
-
         data = {'success': True, 'block': block.to_json_extended()}
+
+        # include_transactions parameter is optional and must be an integer
+        try:
+            include_transactions = parse_int(args.get('include_transactions', 0))
+        except ValueError as e:
+            return json_dumpb({
+                'success': False,
+                'message': f'Failed to parse \'include_transactions\': {e}'
+            })
+
+        if include_transactions == 1:
+            tx_ids: list[str] = []
```

Comment:
Add comment to explain difference between 1 and 2.

Reply:
Also, I guess we can keep 1 and 2 for compatibility, but wouldn't it be better to use something like `include_transactions=txid` (or id/hash/...) and `include_transactions=full` (or metadata/data/meta/...)

Reply:
I just did it. And I guess we don't have to keep compatibility since this is a brand new parameter.